infra/resources/prod/servicebus_topic.tf

```diff
@@ -0,0 +1,101 @@
+/////////////////////////
+//      TOPICS         //
+/////////////////////////
+resource "azurerm_servicebus_topic" "io_auth_topic" {
+  name         = "io-auth-topic"
+  namespace_id = data.azurerm_servicebus_namespace.platform_service_bus_namespace.id
+}
+
+/////////////////////////
+//    SUBSCRIPTIONS    //
+/////////////////////////
+resource "azurerm_servicebus_subscription" "io_auth_sub" {
+  name               = "io-auth-sub"
+  topic_id           = azurerm_servicebus_topic.io_auth_topic.id
+  max_delivery_count = 10
+}
+
+// TODO: add specific filter rules with azurerm_servicebus_subscription_rule
+// terraform resource
+
+/////////////////////////
+//         IAM         //
+/////////////////////////
+
+// Owner role to let the team manage the service bus topic on azure portal
+module "topic_io_auth" {
```

Comment:
Would it be possible to merge this module with `pub_session_manager` and maintain a single `azure-role-assignments` module to manage all team roles?